Dockerfile-dev

```diff
@@ -5,12 +5,12 @@ ARG PACKAGE
 ENV PACKAGE ${PACKAGE}
 COPY . /code
 WORKDIR /code
-RUN echo "building ${PACKAGE}... "; \
-    yarn install; \
-    echo "yarn install done. Building...." ; \
-    yarn build; \
-    echo "building ${PACKAGE} done."; \
+RUN echo "building ${PACKAGE}... " && \
+    yarn install && \
+    echo "yarn install done. Building...." && \
+    yarn build && \
```

Comment:
should this use turbo as well ?

Reply:
Not sure... 
What's the purpose of this Dockerfile anyway?  
It builds the package normal way, then runs `CMD yarn run start:dev:${PACKAGE}`, which is `node esbuild.js --watch & nodemon --watch build --exec node --max-old-space-size=8024 build/index.mjs`, which is watch mode compilation, so `yarn build` gets completely ignored...  
I also don't understand why somebody wants to run watchmode compilation in Docker environment.

Reply:
Will be addressed in https://github.com/w3f/1k-validators-be-PRIVATE/issues/84